node/src/IModelHubDownload.ts

```diff
@@ -3,46 +3,46 @@
 * See LICENSE.md in the project root for license terms and full copyright notice.
 *--------------------------------------------------------------------------------------------*/
 
-import { Logger } from "@bentley/bentleyjs-core";
-import { ElectronAuthorizationBackend } from "@bentley/electron-manager/lib/ElectronBackend";
-import {
-  AuthorizedBackendRequestContext, BriefcaseDb, BriefcaseManager,
-} from "@bentley/imodeljs-backend";
-import { LocalBriefcaseProps } from "@bentley/imodeljs-common";
-import { AccessToken } from "@bentley/itwin-client";
-
+import { BriefcaseDb, BriefcaseManager, IModelHost } from "@itwin/core-backend";
+import { Logger } from "@itwin/core-bentley";
+import { BriefcaseIdValue, LocalBriefcaseProps } from "@itwin/core-common";
+import { NodeCliAuthorizationClient } from "@itwin/node-cli-authorization";
 import { APP_LOGGER_CATEGORY } from "./Main";
 
-// Find your context and iModel IDs at https://www.itwinjs.org/getting-started/registration-dashboard/?tab=1
+// Find your iTwin and iModel IDs at https://developer.bentley.com/my-imodels/
 const IMODELHUB_REQUEST_PROPS = {
-  contextId: "",  // EDIT ME! Specify your own contextId
-  iModelId: "",   // EDIT ME! Specify your own iModelId
+  iTwinId: "", // EDIT ME! Specify your own iTwinId
+  iModelId: "", // EDIT ME! Specify your own iModelId
+};
+
+const AUTH_CLIENT_CONFIG_PROPS = {
+  clientId: "", // EDIT ME! Specify your own clientId
+
+  /** These are the minimum scopes needed - you can leave alone or replace with your own entries */
+  scope: "email openid profile organization itwinjs imodelaccess:read imodels:read",

```

Comment:
The list of scopes needed __should__ only be `imodels:read` scope unless you're doing something else in here.